README.md

```diff
@@ -107,171 +114,173 @@ Server running... press ctrl-c to stop.
 
-### Docker for Mac
+    ```cmd
+    @"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -InputFormat None -ExecutionPolicy Bypass -Command "iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))" && SET "PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin"
+    ```
 
-- Refer [here](https://docs.docker.com/docker-for-mac/install/) for the official installation instructions.
+1.  Verify Chocolatey was added to the environment variables:
 
-### Docker for Windows
+    -  In the Windows UI, open search and type `path`.
+    -  In the Windows CMD console, type `echo %path%`.
+    
+    You should see `C:\ProgramData\chocolatey\bin` in the path.
 
-- Refer [here](https://docs.docker.com/docker-for-windows/install/) for the official installation instructions.
+1.  Close and reopen the command prompt before using `choco` commands.
 
-### Docker Compose
+After running the script at the command line, you can install any required extensions. Chocolatey has many extensions available, similar to Homebrew for MacOS. As a best practice, only use extensions labeled as a "trusted package". You can install editors, such as Nano and Notepad++, using Chocolatey, as well.
 
-- Refer [here](https://docs.docker.com/compose/install/) for the official installation instructions.
+#### Install Ruby extension
 
-### Execution Steps
+If you have Ruby installed on the workstation, then you can skip this section.
 
-1. Using [git](https://git-scm.com/), [clone](https://help.github.com/articles/cloning-a-repository/) this repository.
-2. Navigate to the resulting directory.
-3. Run `docker-compose up` to initialize the build process. Refer [here](https://docs.docker.com/compose/gettingstarted/#step-4-build-and-run-your-app-with-compose) for more details on the use of `docker-compose`.
-4. Visit `http://localhost:4000/` in a web browser, and you should be presented with a local copy of the Magento DevDocs. The configuration for the local port (`4000` by default) is found in the [docker-compose.yml](https://github.com/magento/devdocs/blob/develop/docker-compose.yml) file. If another port is desired, please refer [here](https://docs.docker.com/compose/compose-file/compose-file-v2/#ports) for further details regarding Docker Compose port mapping.
+1.  Open the **Command Prompt** using **Run as Administrator** in the shortcut menu.
 
-### Addressing Problems With Docker Build
+1.  Install the ruby extension:
 
-1. Verify that the Docker engine is installed for the appropriate operating system.
-2. Verify that Docker Compose is installed.
-3. Verify that this repository has been cloned.
-4. Verify that the correct Docker Compose command(s) have been used in the same directory as the `docker-compose.yml` file.
-5. If there are still problems, please open an [Issue](https://help.github.com/articles/creating-an-issue/) on this repository.
+    ```cmd
+    choco install ruby
```

Comment:
this will install the latest Ruby version which is 2.6 that we do not support currently

Reply:
Then put "--version 2.4.3.1" at the end